### PR TITLE
docs: update socials

### PR DESCRIPTION
**Pull request by mpoke**
## Description

Closes: #XXXX

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct `docs:` prefix in the PR title
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] Confirmed the correct `docs:` prefix in the PR title
- [ ] Confirmed all author checklist items have been addressed 
- [ ] Confirmed that this PR only changes documentation
- [ ] Reviewed content for consistency
- [ ] Reviewed content for thoroughness
- [ ] Reviewed content for spelling and grammar
- [ ] Tested instructions (if applicable)

### CODE REVIEW

**Review by MSalopek — APPROVED** [review]

**Review by dusan-maksimovic — APPROVED** [review]

File: docs/docusaurus.config.js [hunk 1]
```diff
@@ -121,20 +121,24 @@ const config = {
...
                 href: "https://forum.cosmos.network/",
-                html: '<i class="fa-fw fa-regular fa-comments"></i> Cosmos Forums',
+                html: '<i class="fa-fw fa-regular fa-comments"></i> Forum',
+              },
+              {
+                href: "https://reddit.com/r/cosmosnetwork",
+                html: '<i class="fa-fw fa-brands fa-reddit"></i> Reddit',
+              },
+              {
+                href: "https://www.youtube.com/c/CosmosProject",
```
Comment (insumity) [inline docs/docusaurus.config.js:140]: Seems last time a video was uploaded there was 2 years ago. Do you want to have this here?

File: docs/docs/delegators/delegator-faq.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ order: 4
 
 ## What is a delegator?
 
-People that cannot or do not want to operate [validator nodes](../validators/overview.mdx) can still participate in the staking process as delegators. Indeed, validators are not chosen based on their self-delegated stake but based on their total stake, which is the sum of their self-delegated stake and of the stake that is delegated to them. This is an important property, as it makes delegators a safeguard against validators that exhibit bad behavior. If a validator misbehaves, their delegators will move their Atoms away from them, thereby reducing their stake. Eventually, if a validator's stake falls under the top 180 addresses with highest stake, they will exit the validator set.
```
Comment (insumity) [inline docs/docs/delegators/delegator-faq.md:8]: Were those links not reachable? If yes, why wasn't this caught by the markdown-link checker?

**Review by insumity — APPROVED** [review]

File: docs/docs/delegators/delegator-faq.md (hunk 2 above)
Comment (mpoke) [reply docs/docs/delegators/delegator-faq.md:8]: The files were changed to .md extension. Not sure about the checker.

File: docs/docusaurus.config.js (hunk 1 above)
Comment (mpoke) [reply docs/docusaurus.config.js:140]: That's a discussion for a different PR.

**Pull request merged**